### ISSUE TITLE
Create some order in "Technology" section

### ISSUE THREAD

**Issue by TomasTomecek** [developer-portal/content#33]
I would sort it alphabetically.

**Comment by vpavlin** [comment]
I agree with @bkabrda that we should sort by popularity

**Comment by bkabrda** [comment]
Yeah, and maybe include search or possibility to sort alphabetically later.

**Comment by TomasTomecek** [comment]
How do you figure out popularity?

**Comment by bkabrda** [comment]
How about counting clicks and sorting the links according to click count? :)

**Comment by vpavlin** [comment]
+1 if we keep the option to sort alphabetically which might be easier for people just scanning what's the content of the site

@asamalik Is something like this (counting clicks, sorting according to those numbers) possible in Jekyll?

**Comment by bkabrda** [comment]
To be more specific: I think it'd be confusing to sort this dynamically. Returning users might see a different order every time, which wouldn't be very user friendly. My advise would be to do an estimate in the beginning and then reorder based on click stats e.g. every half year or so.

**Comment by asamalik** [comment]
I like the idea of sorting content by popularity.

@vpavlin it is not possible. Jekyll generates static pages - so what we deploy is just a set of html files. However, we can use something like Google Analytics which provides very nice overview - even the user-flow through the site.

@bkabrda agree.

I think that Analytics + reordering every half of year or so could do nicely.
